src/flint/fmpz_laurent_series.jl

```diff
@@ -1136,7 +1136,7 @@ function add!(c::ZZLaurentSeriesRingElem, a::ZZLaurentSeriesRingElem, b::ZZLaure
   t = base_ring(a)()
   zc = base_ring(a)()
   for i = 0: lenr - 1
-    pi = valr + sc*i
+    pi = valr + sc*i  # FIXME: there is no sc, perhaps sz is meant? Or zc? Or what?
```

Comment:
I couldn't figure out the correct fix here so I just added a FIXME comments.

Reply:
No idea. Maybe @thofma knows?

Reply:
I vote for `sz`. The code is very similar to https://github.com/Nemocas/AbstractAlgebra.jl/blob/a2e601db066a486140a00a4a732ea050bde2d185/src/generic/LaurentSeries.jl#L1691 and there `sz` is called `sc`.

Reply:
I agree with @joschmitt 's reasoning. For clarity, I would rename the `sz` in line 1124 to `sc` and keep this line as is

Reply:
either is fine I guess, whatever is less work

Reply:
I pushed the change in 57e785e5c33d536cc2d701336d5d437826b4315c